Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:14.04
```

Comment:
It appears the `python`'s base image is Debian-based so `apt` should still be available. However, I am not sure how stripped down they made the image. You could also try this: https://registry.hub.docker.com/u/ipython/scipystack/